lib/Version.php

```diff
@@ -16,5 +16,5 @@ class Version
     /**
      * Full version number.
      */
-    const VERSION = '2.2.2';
+    public const VERSION = '2.2.2';
```

Comment:
This change should not cause any backward-compatibility problem for any consumer that is running at least PHP 7.4. The name and value of the constant is available to consumers just like before.